tests/test_worker.ts

```diff
@@ -710,6 +709,41 @@ describe('workers', function () {
     await worker.close();
   });
 
+
+  describe('when prioritized job is added while processing last active job', () => {
+    it('should process prioritized job whithout delay', async function () {
+      this.timeout(1000);
```

Comment:
why?

Reply:
I copied this test from https://github.com/taskforcesh/bullmq/pull/2176
Did not ask questions about it